snippets/csharp/System.Numerics/BigInteger/.ctor/Project.csproj

```
@@ -0,0 +1,6 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
```

Let's use the latest version:

```suggestion
    <TargetFramework>net9.0</TargetFramework>
```